app/pages/Settings/index.js

```diff
@@ -207,10 +207,10 @@ export default class Settings extends Component {
           () => history.push('/settings/wallet/reveal-seed'),
         )}
         {this.renderSection(
-          'Remove wallet',
-          'This will remove all data from Bob. Proceed with caution.',
```

Comment:
Not sure why but when I run this from `4fb4c1` - `Reset not remove`, I don't see the new copy.
Seems more likely it's local to me, but can you double check?  Is it possible that the Reset props are elsewhere in the codebase, and this new copy isn't actually being loaded?

![Screen Shot 2020-10-16 at 12 17 07 AM](https://user-images.githubusercontent.com/8230144/96224972-37e75680-0f45-11eb-9d78-40b299477304.png)